static/js/invite.js

```diff
@@ -163,17 +164,28 @@ exports.launch = function () {
             hashchange.exit_overlay();
         },
     });
+
+    autosize($("#invitee_emails").trigger("focus"));
+
+    // Ctrl + Enter key to submit form
+    $(document).on("keydown", (e) => {
```

Comment:
We don't want to listen on the whole document but just on the modal container (otherwise this event listener also fires when the modal is closed and you press Ctrl + Enter).

Reply:
I have selected the `invite-user` div now.